x/evm/wrappers/bank.go

```diff
@@ -0,0 +1,111 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package wrappers
```

Comment:
please create a `x/bank/keeper` function and move these files there. We need to keep the same architecture as the IBC module wrapper  because we will need to modify their `module.go` to deprecate the cosmos txs

Reply:
In this case we just need replacement for keeper methods, nothing from the message server. If we want to create a new x/bank module now this PR would change scope. All other `x/<MODULE> `except the `x/ev`m still need the original keeper's methods.

Reply:
Can you add that as a comment to the struct please to avoid future confusion 👍

Reply:
Yeah sure, the idea is to create a good doc. Now it is a simple README because I would like to confirm everything is working good with also the feemarket keeper before spending time to document all the workflow and design decisions!